monai/apps/auto3dseg/auto_runner.py

```diff
@@ -525,7 +525,7 @@ def set_device_info(
         self.device_setting["MN_START_METHOD"] = mn_start_method
 
         if cmd_prefix is None:
-            cmd_prefix = os.environ.get("CMD_PREFIX")
+            cmd_prefix = os.environ.get("CMD_PREFIX", "")
```

Comment:
Thanks @wyli . There seems another occurrence in:

https://github.com/Project-MONAI/MONAI/blob/9f5a55ba479772bb72616aebd23528b9cd63b407/monai/apps/auto3dseg/ensemble_builder.py#L456

Reply:
thanks Mingxin, this is revised now.